test/AuthSamples.FunctionalTests/CustomPolicyProviderTests.cs

```diff
@@ -41,6 +45,69 @@ public CustomPolicyProviderTests(WebApplicationFactory<CustomPolicyProvider.Star
             Assert.Equal("http://localhost/account/signin?ReturnUrl=%2FHome%2FMinimumAge10", response.Headers.Location.ToString());
         }
 
+        [Fact]
+        public async Task MinimumAge10WorksIfOldEnough()
+        {
+            // Arrange & Act
+            var cookie = await SignIn(Client, "Dude", DateTime.Now.Subtract(TimeSpan.FromDays(365 * 20)).ToShortDateString());
```

Comment:
Leap years 😁

Reply:
Shouldn't matter since I'm not doing exactly 10 years, he's 20 years old against a 10 year old check :)